packages/client/src/components/Game/state/state.ts

```diff
@@ -44,13 +45,20 @@ let gameState: IState = {
   score: 0,
   isWinGame: false,
   isGameOver: false,
+  isPause: false,
 }
 
 export const setInitState = (state: IState) => {
   gameState = { ...state }
 }
 
+export const setPause = () => {
+  const pause = gameState.isPause
+  gameState.isPause = !pause
```

Comment:
[Предложение]
тут не нужна дополнительная переменная
```suggestion
  gameState.isPause = !gameState.isPause
```

Reply:
Согласен